jsx/dashboard/config.jsx

```diff
@@ -32,29 +32,54 @@ class config extends React.Component {
   }
 
   save () {
+    let error = false
     let payload = this.state.config
     payload.width = parseInt(payload.width)
     payload.height = parseInt(payload.height)
     payload.column = parseInt(payload.column)
     payload.row = parseInt(payload.row)
-    payload.width = parseInt(payload.width)
-    this.props.updateDashboard(payload)
-    this.setState({updated: false})
+    const fieldsToCheck = ['width', 'height', 'column', 'row']
+    for (let prop of fieldsToCheck) {
+      if (payload[prop] <= 0) {
+        console.log('invalid input')
+        error = true
+      }
+    }
+    if (!error) {
+      this.props.updateDashboard(payload)
+      this.setState({ updated: false })
+    }
   }
 
-  toRow (key, label) {
+  toRow (key, label, Min, Max) {
     let fn = function (ev) {
       var config = this.state.config
-      config[key] = ev.target.value
-      this.setState({
-        updated: true,
-        config: config
-      })
+      let v
+      try {
+        v = parseInt(ev.target.value)
+        if (!isNaN(v) && v <= Max && v >= Min) {
+          config[key] = v
+          this.setState({
+            updated: true,
+            config: config
+          })
+        }
+      } catch (ex) {
+        console.log('invalid input')
```

Comment:
showError ?? Or just drop the console statement ?

Reply:
I think we can just drop the console statement IMO

Reply:
Ok let's drop the console statement. Do we need the try catch block after that?